openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderTrainTest.java

```diff
@@ -287,4 +288,186 @@ private void validateParamsForTheAlgorithm(final Map<String, String> params, fin
                 .as("The list parameter validation errors must be null.")
                 .isEqualTo(0);
     }
+
+    /**
+     * Tests H20 parameter validation for NAIVE_BAYES_CLASSIFIER, all parameters correct.
```

Comment:
use a javadoc link

e.g.
```
Tests H20 parameter validation for {@link H2OAlgorithm#NAIVE_BAYES_CLASSIFIER}, all parameters correct.
```

in IDEA click the name of the method and then click ctrl + q. to have a preview of the javadoc